src/boards/include/boards/solderparty_rp2040_stamp_round_carrier.h

```diff
@@ -36,11 +36,16 @@
 #define PICO_DEFAULT_UART_RX_PIN 17
 #endif
 
-// --- Neopixel ---
+// --- Neopixel Ring ---
 #ifndef PICO_DEFAULT_WS2812_PIN
 #define PICO_DEFAULT_WS2812_PIN 24
 #endif
 
+// --- On-Stamp Neopixel ---
+#ifndef PICO_STAMP_WS2812_PIN
+#define PICO_STAMP_WS2812_PIN 21
```

Comment:
This should probably be something like `SOLDERPARTY_RP2040_STAMP_...` rather than `PICO_STAMP_...`, as it's a feature specific to this board and not to any other Pico boards.

Reply:
Gotcha 👍